wiki/Glossary/en.md

```diff
@@ -275,26 +258,26 @@ The osu!website sorts difficulty versions into into 5 categories:
 
 ### Drain time
 
-The drain time, or "drain", refers to the total duration of the beatmap that requires the player to be active.
+The drain time, or drain, refers to the total duration of the beatmap that requires the player to be active.
```

Comment:
add "Maps with very long drain times are typically referred to as 'marathon' maps and require endurance beyond the norm in order to complete successfully."